srcpkgs/kf6-purpose/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'kf6-purpose'
 pkgname=kf6-purpose
```

Comment:
`kf6-purpose` also depends on optional package KAccounts6 (needed for YouTube and Nextcloud upload).

```sh
hostmakedepends+=" intltool"
makedepends+=" kf6-kaccounts-integration-devel"
```

Reply:
inttools is not required, gettext is already there

Reply:
@sgn can I add `intltool`, I have seen you were removing intltool from many packages before